source/common/common/byte_order.h

```diff
@@ -6,13 +6,70 @@
 
 #include <libkern/OSByteOrder.h>
 
+#define htole16(x) OSSwapHostToLittleInt16((x))
```

Comment:
Can you merge master also when you update? I think @zuercher just added these in his Thrift PR.

Reply:
will do